### PR TITLE
bug-fix: Delete and update comments from issue wasn't working.

### PR DESCRIPTION
**Pull request by joaocsousa**
My first contribution :smile: 

Issue overview: Deleting a comment from an issue wasn't working; Editing a comment from an issue crashed the app.

Current behaviour:

issue_comment_problem

The fragment `IssueFragment` had a `comments` member that was never filled. In my opinion storing comments in both a dedicated `comments` array and in the `adapter` at the same time is not the best option and quickly leads to inconsistency problems.

New behaviour:

issue_comment_fix

Comments are welcome :wave:

### CODE REVIEW

File: app/src/main/java/com/github/pockethub/ui/issue/DeleteCommentTask.java [hunk 1]
```diff
@@ -62,7 +62,7 @@ public DeleteCommentTask(final Context context,
 
     @Override
     protected GithubComment run(Account account) throws Exception {
-        new DeleteIssueCommentClient(context, InfoUtils.createRepoInfo(repository), comment.id);
+        new DeleteIssueCommentClient(context, InfoUtils.createRepoInfo(repository), comment.id).executeSync();
```
Comment (Meisolsson) [inline app/src/main/java/com/github/pockethub/ui/issue/DeleteCommentTask.java]: I feel like a idiot for the moment... need to be more awake when re-writing code

Comment (larsgrefer) [inline app/src/main/java/com/github/pockethub/ui/issue/DeleteCommentTask.java]: Once we upgrade to Retrofit 2.0 this becomes better :wink:

**Comment by larsgrefer** [comment]
@joaocsousa How did you create the gif's?

**Comment by joaocsousa** [comment]
@larsgrefer genymotion (or the normal android emulator, wtv you prefer) to run the app, plus licecap (available for windows and mac) for screen grabbing to gif.

**Comment by Meisolsson** [comment]
Looks good.
Thanks for the fix!

**Pull request merged**